pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -10,6 +10,10 @@
 , libXext
 , libXfixes
 , ncurses
+, xorg
+, libgudev
+, wavpack
+, jack2
```

Comment:
Jack adds a dependency on Qt through ffado :crying_cat_face:

Reply:
OK, made it optional.